pkg/agent/config/node_config.go

```diff
@@ -44,6 +44,11 @@ const (
 	IPv6ExtraOverhead = 20
 )
 
+const (
+	L7NetworkPolicyTargetPortName = "antrea-tap0"
```

Comment:
Should we use meaningful names like "antrea-l7-tap", "antrea-l7-ret-tap"? @tnqn

Reply:
If we are clear there will only be two ports created for L7, I feel something like `antrea-l7-rev-tap`, `antrea-l7-ret-tap` are ok. But I am also ok with antrea-l7-tap0, antrea-l7-tap1.